generated.go

```diff
@@ -2,14 +2,14 @@
 package ipfs
 
 import (
-	"context"
-	"io"
-
-	"github.com/beyondstorage/go-storage/v4/pkg/credential"
-	"github.com/beyondstorage/go-storage/v4/pkg/endpoint"
-	"github.com/beyondstorage/go-storage/v4/pkg/httpclient"
-	"github.com/beyondstorage/go-storage/v4/services"
-	. "github.com/beyondstorage/go-storage/v4/types"
+    "context"
+    "io"
+
+    "github.com/beyondstorage/go-storage/v4/pkg/endpoint"
+    "github.com/beyondstorage/go-storage/v4/pkg/credential"
+    "github.com/beyondstorage/go-storage/v4/pkg/httpclient"
+    "github.com/beyondstorage/go-storage/v4/services"
+    . "github.com/beyondstorage/go-storage/v4/types"
```

Comment:
It seems that this file is not formatted. You can use `make build` where `go fmt` is included.